### PR TITLE
Fixes for modular generation

### PR DESCRIPTION
**Pull request by kuhnroyal**
This fixes 2 problems with modular generation:
- fix duplicate result set classes being generated in a single accessor
- fix type converters missing import prefixes

### CODE REVIEW

File: drift_dev/lib/src/writer/writer.dart [hunk 1]
```diff
@@ -198,7 +198,9 @@ abstract class _NodeOrWriter {
     final buffer = StringBuffer();
 
     for (final lexeme in code.elements) {
-      if (lexeme is DartTopLevelSymbol) {
+      if (lexeme is AnnotatedDartCode) {
```
Comment (simolus3) [inline drift_dev/lib/src/writer/writer.dart]: An `AnnotatedDartCode` shouldn't have another `AnnotatedDartCode` as an element, that's probably a bug in another part of the generator.

I think we should add an assertion to the constructor of `AnnotatedDartCode` checking that each element is either a `String` or a `DartTopLevelSymbol`. Then we can figure out where the wrong element comes from and fix that.

**Review by simolus3 — COMMENTED** [review]
Thanks for the fixes! I'm a bit worried about `AnnotatedDartCode` apparently nesting - if you have an example where that happens I can also take a look at fixing that.

File: drift_dev/lib/src/writer/writer.dart (hunk 1 above)
Comment (kuhnroyal) [reply drift_dev/lib/src/writer/writer.dart]: That happens when the type converter is read: https://github.com/simolus3/drift/blob/develop/drift_dev/lib/src/writer/writer.dart#L115
And this results in the `AnnotatedDartCode.toString()` being used which is missing the import prefix.

**Comment by kuhnroyal** [comment]
Yea that makes more sense, I run a quick test.

**Comment by kuhnroyal** [comment]
Works on my project 👍

**Review by simolus3 — APPROVED** [review]
Thanks again for the fixes!

**Pull request merged**